### PR TITLE
Fix documentation for programmatic REST Client Reactive registration

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: docs/src/main/asciidoc/rest-client-reactive.adoc [hunk 1]
```diff
@@ -286,7 +287,7 @@ public class ExtensionsResource {
 
     public ExtensionsResource() {
         extensionsService = RestClientBuilder.newBuilder()
-            .baseUri("https://stage.code.quarkus.io/api")
+            .baseUri(URI.create("https://stage.code.quarkus.io/api"))
```
Comment (gsmet) [inline docs/src/main/asciidoc/rest-client-reactive.adoc:290]: I wonder if we should add a way to use a string instead. But maybe it's a bad idea.

**Review by gsmet — APPROVED** [review]

**Comment by michalszynkiewicz** [comment]
Thanks Georgios!

File: docs/src/main/asciidoc/rest-client-reactive.adoc (hunk 1 above)
Comment (geoand) [reply docs/src/main/asciidoc/rest-client-reactive.adoc:290]: It sounds like a good idea to me, but the user would have to downcast to get it

**Comment by geoand** [comment]
YW!

**Pull request merged**